wolfcrypt/src/hmac.c

```diff
@@ -341,6 +341,11 @@ int wc_HmacSetKey(Hmac* hmac, int type, const byte* key, word32 length)
     hmac->keyLen = length;
 #endif
 
+#ifdef WOLFSSL_MAXQ108X
+    /* For MAXQ108x, nothing left to do. */
+    return 0;
+#endif
+
```

Comment:
Good idea. 

I originally did it this way to minimize the scope of the change and because I thought the compiler would optimize away everything after the return, but your suggestion is clearer.